client/.env.example

```diff
@@ -24,5 +24,7 @@ DISCORD_CLIENT_SECRET=""
 FIREBASE_API_KEY=AIzaSyB2ibR4-0Zu1I1rODsLQSWqm-sBt9IUjzE
 
 #Pinata
-PINATA_GATEWAY_TOKEN=hvd5WS5pWXPE_EzbvBRqeT8dk-4lZ3FRoLsav7ibt7p70Ici9GMC9BrM2uzJMdXA
-PINATA_BEARER_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySW5mb3JtYXRpb24iOnsiaWQiOiJhODc0OGJjMC1jMWJlLTRiODctOTc2NS03N2Y0NjdkMjJlOTkiLCJlbWFpbCI6Imthcm9sQHNsaWNlZC5jbHViIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsInBpbl9wb2xpY3kiOnsicmVnaW9ucyI6W3siaWQiOiJGUkExIiwiZGVzaXJlZFJlcGxpY2F0aW9uQ291bnQiOjF9LHsiaWQiOiJOWUMxIiwiZGVzaXJlZFJlcGxpY2F0aW9uQ291bnQiOjF9XSwidmVyc2lvbiI6MX0sIm1mYV9lbmFibGVkIjpmYWxzZSwic3RhdHVzIjoiQUNUSVZFIn0sImF1dGhlbnRpY2F0aW9uVHlwZSI6InNjb3BlZEtleSIsInNjb3BlZEtleUtleSI6ImJmYzFkNTgwZmJhMDY5NWUwMDEwIiwic2NvcGVkS2V5U2VjcmV0IjoiOGRjYWQ4YjI3YzdlY2JiNTgwZDQ0NjkzNGY3MTJlYjU1MDc2OTRiZDFmNDQ1Mzc4NDUxZjc3YjA3ZTc5YzllYSIsImlhdCI6MTcxMTQ1MzQ2MH0.mWD-_slO8jVy1vZERjLe4nSljdLw4Q2kiHinpwcVxQE
+PINATA_GATEWAY_URL="tan-late-louse-578.mypinata.cloud"
+PINATA_API_TOKEN=619c888609cb4ae11a1c
+PINATA_SECRET_API_TOKEN=90aa90445a9f4849cefd5d003922d91d1d914b899c42404df3668cf26d84e14b
+PINATA_BEARER_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySW5mb3JtYXRpb24iOnsiaWQiOiJhODc0OGJjMC1jMWJlLTRiODctOTc2NS03N2Y0NjdkMjJlOTkiLCJlbWFpbCI6Imthcm9sQHNsaWNlZC5jbHViIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsInBpbl9wb2xpY3kiOnsicmVnaW9ucyI6W3siaWQiOiJGUkExIiwiZGVzaXJlZFJlcGxpY2F0aW9uQ291bnQiOjF9LHsiaWQiOiJOWUMxIiwiZGVzaXJlZFJlcGxpY2F0aW9uQ291bnQiOjF9XSwidmVyc2lvbiI6MX0sIm1mYV9lbmFibGVkIjpmYWxzZSwic3RhdHVzIjoiQUNUSVZFIn0sImF1dGhlbnRpY2F0aW9uVHlwZSI6InNjb3BlZEtleSIsInNjb3BlZEtleUtleSI6IjYxOWM4ODg2MDljYjRhZTExYTFjIiwic2NvcGVkS2V5U2VjcmV0IjoiOTBhYTkwNDQ1YTlmNDg0OWNlZmQ1ZDAwMzkyMmQ5MWQxZDkxNGI4OTljNDI0MDRkZjM2NjhjZjI2ZDg0ZTE0YiIsImlhdCI6MTcxMjM1NjcyNX0.G68HuRmrSJv19AXVzvyrSeifK3QFfJuC_AsaeAb4xQg
```

Comment:
I'm sure you're aware but best to not have the actual keys in a public repo

Reply:
Yeah man, I acc forgot this was a public repo but env.example is meant so that devs working on this can all see the keys at all times. Taken from the T3 setup:

_Instead, we recommend that you store secrets in .env, keep your .env file in your .gitignore and only commit .env.example files to your project._

I'll remove the keys from .env.example.